src/njoy21/input/DTFR/Card4/HollerithNames/test/HollerithNames.test.cpp

```diff
@@ -0,0 +1,58 @@
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "HollerithNames input values",
+         "[Card4], [HollerithNames]" ){
+  Argument< DTFR::Card3::Iptotl > iptotl;
+  iptotl.value = 6 ;
+
+ GIVEN( "the correct number of names" ){
+
+    WHEN( "the names are of correct length" ){
+      iRecordStream< char> iss( std::istringstream("pheat1 edits dtfr") );
```

Comment:
The manual says they have to be "delimited" by an apostrophe. Does that mean "surrounded by"? I did think it was odd. If the values are simply surrounded by them, then we are probably okay.